frontend/src/locales/de.json

```diff
@@ -6,7 +6,8 @@
     },
     "safetyConcept": {
       "add": "Sicherheits-Konzept hinzufügen",
-      "name": "Sicherheits-Konzept"
+      "name": "Sicherheits-Konzept",
+      "sort": "3"
```

Comment:
Für was wird das benötigt?

Reply:
Ich dachte es ist für die Reihenfolge der ActivityContents, habs aber nicht verifiziert. Würde ich auch gleich mit #580 aus den Übersetzungen entfernen und in andersweitige Konfiguration auslagern.